bin/setup_db

```diff
@@ -83,6 +83,9 @@ def setup_db(host, port, name):
         LOGGER.info('Creating table: queue_outbound')
         r.db(name).table_create('queue_outbound').run(conn)
 
+        LOGGER.info('Creating table: queue_outbound_complete')
+        r.db(name).table_create('queue_outbound_complete').run(conn)
```

Comment:
What is the purpose of this queue? I am asking in hopes of keeping the names making sense as more are added.

Reply:
Makes sense. I'll change it

Reply:
Adam created an issue a while back saying we needed a changelog. This is a simple naive way to keep one for now until we can find a better solution and refactor.

Reply:
Sure, a table works as a makeshift changelog in the interim. We can redirect the stream later when we have an alternative logging solution in place.